File: VB/Lib/CMCommon/Common/CMCommonSelectArgs.vb
Namespace Common
    ''' <summary>
    ''' 共通検索呼び出し用引数データ
    ''' </summary>
    Public Class CMCommonSelectArgs
#Region "プロパティ"
        ''' <summary>検索ID</summary>
        Public Property SelectId As String

        ''' <summary>パラメータ</summary>
        Public Property Params As Object()
#End Region

#Region "コンストラクタ"
        ''' <summary>
        ''' コンストラクタ
        ''' </summary>
        ''' <param name="argSelectId">検索ID</param>
        ''' <param name="argParams">パラメータ</param>
        Public Sub New(argSelectId As String, ParamArray argParams As Object())
            SelectId = argSelectId
            Params = argParams
        End Sub
#End Region
    End Class
End Namespace

File: CS/Lib/CMCommon/Common/CMCommonSelectArgs.cs
/*******************************************************************************
 * 【共通部品】
 *
 * 作成者: 日進テクノロジー／田中 望
 * 改版履歴:
 * 2014.1.30, 新規作成
 ******************************************************************************/
using System;
using System.Collections.Generic;
using System.Text;

namespace NEXS.ERP.CM.Common
{
    //************************************************************************
    /// <summary>
    /// 共通検索呼び出し用引数データ
    /// </summary>
    //************************************************************************
    [Serializable]
    public class CMCommonSelectArgs
    {
        #region プロパティ
        /// <summary>検索ID</summary>
        public string SelectId { get; set; }

        /// <summary>パラメータ</summary>
        public object[] Params { get; set; }
        #endregion

        #region コンストラクタ
        //************************************************************************
        /// <summary>
        /// コンストラクタ
        /// </summary>
        /// <param name="argSelectId">検索ID</param>
        /// <param name="argParams">パラメータ</param>
        //************************************************************************
        public CMCommonSelectArgs(string argSelectId, params object[] argParams)
        {
            SelectId = argSelectId;
            Params = argParams;
        }
        #endregion
    }
}
